README.md

```diff
@@ -63,8 +63,10 @@ The goal of this project is to make the easiest, fastest, and most painless way
 **Note: As Gitea is a [Gogs](https://github.com/gogits/gogs) fork, tutorials and documentation related to gogs applies to Gitea too**
 
 How to install Gitea:
+_Do note that this Gitea branch is **not** stable enough for builds_
```

Comment:
@LefsFlarey are you planning to accept the change suggestion here ? Hanging PRs make me anxious :)